libs/blocks/global-navigation/features/breadcrumbs/breadcrumbs.js

```diff
@@ -85,7 +85,10 @@ const fromUrl = () => {
   if (getMetadata(metadata.fromUrl) !== 'on') return null;
   const list = toFragment`<ul></ul>`;
   const paths = document.location.pathname.split('/').filter((n) => n);
+  const locales = getConfig().locales || {};
   for (let i = 0; i < paths.length; i += 1) {
+    // eslint-disable-next-line no-continue
+    if (locales[paths[i]]) continue;
```

Comment:
I'm not sure I want to disable the `no-continue` rule for this. Initially I was going to suggest we simply wrap the logic inside an `if` clause, but then I figured: why do this check for all breadcrumb entries? The locale should be the first one, right? So how about: outside the `for` loop, check whether the first path matches a known locale. If so, initialise `i` as `1` instead of `0`.

Reply:
Is it guaranteed to be the first path? In theory, it could as well be `stories/ch_de/...` although if a consumer opted for that - that might have other side effects

Reply:
The general rule has the locale as the first item, so that sounds like a good bet to me.

Another question: doesn't getConfig have a `prefix` property under the `locale` object? Maybe we can use just that and do a string replace in the initial `document.location.pathname`, before doing the split

Reply:
Yeah bunch of ways to solve this, prefix sounds good 👍